Common/Brokerages/GDAXBrokerageModel.cs

```diff
@@ -0,0 +1,104 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using QuantConnect.Data.Market;
+using QuantConnect.Orders;
+using QuantConnect.Securities;
+using QuantConnect.Securities.Interfaces;
+using QuantConnect.Orders.Fills;
+using QuantConnect.Orders.Fees;
+using QuantConnect.Orders.Slippage;
+using QuantConnect.Configuration;
+using System.Linq;
+using QuantConnect.Logging;
+
+namespace QuantConnect.Brokerages
+{
+
+    /// <summary>
+    /// Provides GDAX specific properties
+    /// </summary>
+    public class GDAXBrokerageModel : DefaultBrokerageModel
+    {
+
+        private static BrokerageMessageEvent _message = new BrokerageMessageEvent(BrokerageMessageType.Warning, 0, "Brokerage does not support update. You must cancel and re-create instead.");
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="GDAXBrokerageModel"/> class
+        /// </summary>
+        /// <param name="accountType">The type of account to be modelled, defaults to 
+        /// <see cref="QuantConnect.AccountType.Margin"/></param>
+        public GDAXBrokerageModel(AccountType accountType = AccountType.Margin)
+            : base(accountType)
+        {
+
+        }
+
+        /// <summary>
+        /// GDAX global leverage rule
+        /// </summary>
+        /// <param name="security"></param>
+        /// <returns></returns>
+        public override decimal GetLeverage(Security security)
+        {
+            return 3m;
```

Comment:
Perhaps 1 to start? By memory you mentioned in chat the code doesn't support leverage yet?

Reply:
I've not been able to test leverage. In theory nothing remains to be done to support leverage, but it sounds like we will need to disable for now.